Lectures/Lecture3_BigCollaborations.md

```diff
@@ -1,41 +1,62 @@
+This lecture focuses on large national as well as international collaborations. Many of the large collaborations involve interdisciplinary components, both within physics and with engineering and industry. See https://bigscience.nl/about for a concerted effort in NL to foster synergies between physics and technology in large collaborations. 
 
 ## Methods/exercises
-**Topics covered in lecture**
-+ topic 1
-+ topic 2
++ In preparation for this lecture, course participants are asked to watch the two-part video lecture by Dr Niels Martens on the epistemology of scientific collaborations. Lecture 1 of this set (https://www.youtube.com/watch?v=M1giIxeLuIg) discusses examples of 'big science' projects such as CERN and the Next Generation Event Horizon Telescope. Lecture 2 of this set (https://www.youtube.com/watch?v=DRt1uTytWf8) focuses on ways to manage the large number of co-authors that are involved in these projects and policies for giving appropriate credit. 
++ Course participants will work in small groups to investigate the credit structure of one collaboration they were assigned from the list below. They prepare a short presentation to their peers. 
++ The groups will give their presentations to the class and have a final critical discussion on the topic of large collaborations, for which Dr. Niels Martens will also join. 
 
 ## Evaluation
-__assignments or evaluation of the desired learning outcome__
++ Students answer the quiz about the video lectures by Dr. Niels Martens. 
++ Students work in small groups to investigate an example of the credit structure and promotion of junior researchers within a large collaboration.
++ Students give a short presentation of their findings to the class, receive peer feedback, and hear from the others on different examples.  
++ Students participate in the final discussion. 
 
 ### Assignment description
 
+Students work in small groups to investigate the credit structure of one of the large collaborations from the list below. They prepare a short presentation that addresses:
+
+1. A brief introduction to the collaboration, its scientific aims, when it was established, and its membership. 
+2. An assessment of how credit structures are implemented and how junior researchers can grow within this big collaboration and gain visibility in the field, as far as can be determined from the resources that are made publicly available by the collaboration. 
+
+List of example collaborations to investigate:
+- Intergovernmental Panel on Climate Change  http://www.ipcc.ch
+- ALICE experiment at CERN  https://alice-collaboration.web.cern.ch
+- Einstein Telescope collaboration https://www.et-gw.eu/index.php/the-et-collaboration
+- Pangeo https://pangeo.io
+- Euclid consortium https://www.euclid-ec.org 
+- European Centre for Medium Range Weather Forecasts  https://www.ecmwf.int
+- Quantum Delta NL https://quantumdelta.nl
+- Cosmic-Ray Pierre Auger collaboration https://www.auger.org/collaboration 
+
```

Comment:
Another option could be NESSC (https://www.nessc.nl)?

Reply:
good, added this